frontend/src/features/workspace/components/AdminFeedbackBox/AdminFeedbackBox.tsx

```diff
@@ -109,20 +115,18 @@ const AdminFeedbackCallForCommentContent = ({
   onLinkClick: () => void
   ratingValue: AdminFeedbackRating
 }) => {
-  // Thumbs down
-  if (ratingValue === AdminFeedbackRating.down)
-    return (
-      <Text textStyle="h6">
-        Thank you for your feedback.{' '}
-        <Link onClick={onLinkClick}>Tell us more so we can improve!</Link>
-      </Text>
-    )
-
-  // Thumbs Up
+  const ratingName = AdminFeedbackRating[
+    ratingValue
+  ] as keyof typeof AdminFeedbackRating
+  const { t } = useTranslation()
+  const { title, link } = t(
+    `features.workspace.feedback.callForComment.${ratingName}`,
```

Comment:
Try to avoid string interpolation for translations. Makes it very difficult to debug.

Reply:
hmm, but I think using a tenary (or other conditional) to determine the key to be used is quite brittle. I don't mind going with that if that's what you prefer. Or is there yet a better way?